### DISCUSSION TITLE
how to show my email id in my github profile

### DISCUSSION THREAD

**Discussion by AthulOntop** [community/community#19708] [Other Feature Feedback, Questions, & Ideas] [answered]
please help

**Comment by sanrionpc** [comment, marked as answer]
show the link: https://stackoverflow.com/questions/62377529/is-it-possible-to-show-an-email-on-github-profile-while-keeping-other-emails-pri

**Comment by alkock** [comment]
Go to the top right corner -> Press on your Profile-Icon -> go to settings -> under Access & E-Mail you'll find the related settings
